README.md

```diff
@@ -19,15 +19,6 @@ const Resolver = EmberResolver.extend(HotReloadMixin);
 export default Resolver;
 ```
 
-Add the development settings for ember-devtools to your config/environment.js
```

Comment:
I only removed this and not the other reference to devtools ^^ to avoid conflicts with PR#2